### PR TITLE
SOC-3020 |  Community Avatar and Discussions Header disappear on error pages

### PR DESCRIPTION
**Pull request by mleczey**
## Links

* https://wikia-inc.atlassian.net/browse/SOC-3020

## Description

Community Avatar and Discussions Header disappear on error pages.

## Reviewers

@bkoval

### CODE REVIEW

File: front/main/app/models/discussion/site-attributes.js [hunk 1]
```diff
@@ -71,18 +71,20 @@ DiscussionSiteAttributesModel.reopenClass({
...
 			const attributesInstance = DiscussionSiteAttributesModel.create();
 
 			attributesInstance.set('wikiId', wikiId);
 
 			request(M.getAttributeServiceUrl(`/site/${wikiId}/attr`)).then((data) => {
 				attributesInstance.setNormalizedData(data);
+
+				resolve(attributesInstance);
 			}).catch((err) => {
 				attributesInstance.setErrorProperty(err);
```
Comment (bkoval) [inline front/main/app/models/discussion/site-attributes.js]: Do we still need this code? Seems like something that should be handled automatically by the error substate.

File: front/main/app/routes/discussion/error.js [hunk 2]
```diff
@@ -1,3 +1,10 @@
 import DiscussionBaseRoute from './base';
 
-export default DiscussionBaseRoute.extend({});
+export default DiscussionBaseRoute.extend({
+	setupController(controller, error) {
+		const discussionModel = this.modelFor('discussion');
+
+		this._super(...arguments);
+		error.index = discussionModel;
```
Comment (bkoval) [inline front/main/app/routes/discussion/error.js]: Error.set?

File: front/main/app/models/discussion/site-attributes.js (hunk 1 above)
Comment (mleczey) [inline front/main/app/models/discussion/site-attributes.js]: Ticket to investigate this was created: SOC-3035

Comment (bkoval) [inline front/main/app/models/discussion/site-attributes.js]: 💛

**Comment by bkoval** [comment]
LGTM 👹

**Pull request merged**